### ISSUE TITLE
Folder name sanitize options

### ISSUE THREAD

**Issue by lucalanca** [gmetais/grunt-devperf#21]
I recently had the need to increase the maximum number of characters in the folder name sanitize function (https://github.com/gmetais/grunt-devperf/blob/master/tasks/devperf.js#L292). (really long url as it is my testing environment).

I forked and pointed my project to my forked version however, I would like to still point to the main repo. Can we increase this size? Or make it configurable as a grunt option? Or even better by using a method that tries to make it as short as possible, e.g.:

```
PATH/foo/bar/bar.html
PATH/foo/bar/foo.html
PATH/foo/foo/bar.html
```

that would create

```
bar-bar
bar-foo
foo-bar
```

basically removing the protocol and domain part, plus the common path between all urls.
